src/hooks/mySql.js

```diff
@@ -32,11 +32,26 @@ function extractQueryFromArg(arg) {
   }
 }
 
+function extractValuesFromArg(arg) {
+  if (isObject(arg) && typeof arg === 'object' && !!arg['values']) {
+    return arg['values'];
+  }
+}
+
 function queryBeforeHook(args, extenderContext) {
   const awsRequestId = TracerGlobals.getHandlerInputs().context.awsRequestId;
   const transactionId = getCurrentTransactionId();
   const query = extractQueryFromArg(args[0]);
-  const values = Array.isArray(args[1]) ? args[1] : [];
+
+  const values =
+    // This is for the `.query(options, callback?)` form
+    extractValuesFromArg(args[0]) ||
```

Comment:
i would argue that this method should be named `extractValuesFromQueryOptions` and that the second "half" of the ternary be extracted to a method named `extractValuesFromQueryParameter`

Reply:
Fixed in 5fbb948901e88348c95947aa27909090528b9fde